aries-site/src/pages/components/namevaluelist.mdx

```diff
@@ -155,7 +155,14 @@ In some cases, NameValuePairs may be used as read-only display for text inputs t
 
 ### Content Truncation
 
-When a value content exceeds 192 characters, truncate content and include a Button to expand to see more.
+When presenting a value as a list of items, the list may be truncated and 
+accompanied by a Button to expand it, displaying all items.
+
+When delimited, content exceeding 192 characters should be truncated. 
+
+For vertical lists:
```

Comment:
Should we provide any guidance about when to split with a "," vs stacking vertically? Right now it looks like when it's plain text it should be a comma delimited list, but when it's an Anchor is can be vertically stacked but I'm not sure that's the guidance. Is it up to the team to use their discretion to decide? @bahriaditi

Reply:
Great point @taysea! From a UX perspective, I don't have enough context to provide that design guidance just yet. It'll be interesting to see new use cases that might come up, so we can start to see a pattern emerging. For the time-being, though, I'd say its up to the designer's discretion.

Reply:
pulling from Slack discussion: "short answer is - i think we should leave it to the team's discretion until we see more use cases emerging"